site/operational-guide/tasks.md

```diff
@@ -0,0 +1,102 @@
+  reset` - for example due to a hardware failure or other unplanned
+  termination
+
+In some cases you may already be aware of the problem, as you were
+unable to execute `weave reset` successfully or because you know
+through other channels that the host has died - in these cases you can
+proceed straight to the section on reclaiming lost space.
+
+However in some scenarios (for example when using autoscaling and
+spot-instances that can be destroyed without notice) it may not be
+obvious that space has been lost, in which case you can check for it
+periodically with the following command on any peer:
+
+    weave status ipam
+
+This will list the names of unreachable peers; if you are satisifed
+that they are truly gone, rather than temporarily unreachable due to a
+partition, you can reclaim their space by following the advice in the
+next section.
+
+## Reclaiming Lost IP Address Space
+
+When a peer dies unexpectedly the remaining peers will consider its
+address space to be unavailable even after it has remained unreachable
+for prolonged periods; there is no universally applicable time limit
+after which one of the remaining peers could decide unilaterally that
+it is safe to appropriate the space for itself, and so an
+administrative action is required to reclaim it.
+
+The `weave rmpeer` command is provided to perform this task, and must
+be executed on one of the remaining peers. That peer will take
+ownership of the freed address space.
+
+## Rolling Upgrades
+
+Protocol versioning and feature negotiation are employed in Weave Net
+to enable incremental rolling upgrades - each major release maintains
+the ability to speak to the preceeding major release at a minimum, and
+connected peers only utilise features which both support. The general
+upgrade procedure is as follows:
+
+On each peer in turn:
+
+* Stop the old weave with `weave stop` (or `systemctl stop weave` if
+  you're using a systemd unit file)
+* Download the new weave script and replace the existing one
+* Start the new weave with `weave launch <existing peer list>` (or
+  `systemctl start weave` if you're using a systemd unit file)
+
+This will result in some downtime as the first launch with the new
+script has to pull the new container images; if you wish to minimise
+downtime you can download the new script to a temporary location
+first:
+
+* Download the new weave script to a temporary location e.g.
+  `/path/to/new/weave`
+* Pull the new images with `/path/to/new/weave setup`
+* Stop the old weave with `weave stop` (or `systemctl stop weave` if
+  you're using a systemd unit file)
+* Replace the existing script with the new one
+* Start the new weave with `weave launch <existing peer list>` (or
+  `systemctl start weave` if you're using a systemd unit file)
+
+> NB Always check the release notes for specific versions in case
+> there are any special caveats or deviations from the standard
+> procedure.
+
+## Resetting Persisted Data
```

Comment:
This is a bit obscure as a heading in that you need to know quite a lot about weave in order to know why you'd want to do this. So either try to capture the most important of these in the heading, or leave that section out (and expand on descriptions of 'weave reset' elsewhere, and then reference that from other places).